src/components/vacation-requests-table/vacation-requests-table-toolbar/toolbar-update-status-button.tsx

```diff
@@ -12,9 +12,8 @@ interface UpdateStatusButtonProps {
     newStatus: VacationRequestStatuses,
     selectedRowIds: GridRowId[]
   ) => Promise<void>;
-  approval: boolean;
+  approval?: boolean;
```

Comment:
What about changing this to something like `buttonType: VacationRequestStatuses`?

Then within `::handleUpdateVacationRequestStatus` you could do something like:

```typescript
  const handleUpdateVacationRequestStatus = async () => {
    await updateVacationRequestStatuses(buttonType, selectedRowIds);
  };
```